persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,147 @@
+        Assert.assertEquals(null, structuralVariantFirstResult.getSite2EffectOnFrame());
+        Assert.assertEquals("no", structuralVariantFirstResult.getDnaSupport());
+        Assert.assertEquals("yes", structuralVariantFirstResult.getRnaSupport());
+        Assert.assertEquals(null, structuralVariantFirstResult.getNormalReadCount());
+        Assert.assertEquals((Integer) 100000, structuralVariantFirstResult.getTumorReadCount());
+        Assert.assertEquals(null, structuralVariantFirstResult.getNormalVariantCount());
+        Assert.assertEquals((Integer) 100000, structuralVariantFirstResult.getTumorVariantCount());
+        Assert.assertEquals(null, structuralVariantFirstResult.getNormalPairedEndReadCount());
+        Assert.assertEquals(null, structuralVariantFirstResult.getTumorPairedEndReadCount());
+        Assert.assertEquals(null, structuralVariantFirstResult.getNormalSplitReadCount());
+        Assert.assertEquals(null, structuralVariantFirstResult.getTumorSplitReadCount());
+        Assert.assertEquals("AKT1-AKT2", structuralVariantFirstResult.getAnnotation());
+        Assert.assertEquals(null, structuralVariantFirstResult.getBreakpointType());
+        Assert.assertEquals(null, structuralVariantFirstResult.getCenter());
+        Assert.assertEquals(null, structuralVariantFirstResult.getConnectionType());
+        Assert.assertEquals("Fusion", structuralVariantFirstResult.getEventInfo());
+        Assert.assertEquals(null, structuralVariantFirstResult.getVariantClass());
+        Assert.assertEquals(null, structuralVariantFirstResult.getLength());
+        Assert.assertEquals("Gain-of-Function", structuralVariantFirstResult.getComments());
+        Assert.assertEquals("Genbank:AB209510", structuralVariantFirstResult.getExternalAnnotation());
+        Assert.assertEquals(null, structuralVariantFirstResult.getDriverFilter());
+        Assert.assertEquals(null, structuralVariantFirstResult.getDriverFilterAnn());
+        Assert.assertEquals(null, structuralVariantFirstResult.getDriverTiersFilter());
+        Assert.assertEquals(null, structuralVariantFirstResult.getDriverTiersFilterAnn());
+
+    }
+
+    @Test
+    public void fetchStructuralVariantsWithSampleIdentifiers() throws Exception {
+
+        List<String> geneticProfileStableIds = new ArrayList<String>();
+        List<String> hugoGeneSymbols = new ArrayList<String>();
+        List<String> studyIds = new ArrayList<String>();
+        List<String> sampleIds = new ArrayList<String>();
+
+        geneticProfileStableIds.add("study_tcga_pub_sv");
+        hugoGeneSymbols.add("AKT1");
+        studyIds.add("acc_tcga");
+        sampleIds.add("TCGA-A1-B0SO-01");
+
+        List<StructuralVariant> result = 
+                structuralVariantMyBatisRepository.fetchStructuralVariants(geneticProfileStableIds, 
+                        hugoGeneSymbols, studyIds, sampleIds);
+
+        Assert.assertEquals(2,  result.size());
```

Comment:
ideally this will be different from previous test, so that you can check if the extra parameter is actually filtering anything.